packages/dataviews/src/constants.ts

```diff
@@ -46,10 +46,12 @@ export const OPERATORS = {
 	},
 };
 
-// Sorting
-export const SORTING_DIRECTIONS = {
-	asc: { label: __( 'Sort ascending' ) },
-	desc: { label: __( 'Sort descending' ) },
+export const SORTING_DIRECTIONS = [ 'asc', 'desc' ] as const;
```

Comment:
Why we do the const assertions in some of these values and not for all? Is there a specific reason?

Reply:
yes, there is and it's something I learned from @sirreal's commit description here https://github.com/WordPress/gutenberg/pull/61682/commits/d5f941eb94a9fba8389fbcfd48e5f65582e6a5ad

Basically it forces typescript to understand that these are constant types and not variable string type.